packages/ra-core/src/controller/create/useCreateController.ts

```diff
@@ -1,8 +1,7 @@
 import { useCallback } from 'react';
 // @ts-ignore
 import { parse } from 'query-string';
-import { useLocation } from 'react-router-dom';
-import { Location } from 'history';
+import { useLocation, Location } from 'react-router-dom';
```

Comment:
👍 I believe the codemod didn't catch this one, am I right?

Reply:
Actually the codemod removes all imports from `history`, not only `createMemoryHistory`, so I saw it thanks to a combination of the codemod and TypeScript complaining about the type from `Location` being incorrect.